src/galaxy/config.py

```diff
@@ -21,12 +21,20 @@
 
 from configparser import ConfigParser
 import logging
+import os
+from slowapi.util import get_remote_address
+from slowapi import Limiter
 
 CONFIG_FILE_PATH = "src/config.txt"
 
+if os.path.exists(CONFIG_FILE_PATH) is False:
+    raise "Config file does not exist : src/config.txt"
```

Comment:
```
  File "/home/nicolas/Documents/GitHub/galaxy-api/./src/galaxy/config.py", line 31, in <module>
    raise "Config file does not exist : src/config.txt"
TypeError: exceptions must derive from BaseException
```
- I think you should raise an Exception with a str in param instead of directly a str.
- It should be better yo use CONFIG_FILE_PATH in the error str